modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/portlet/SamlAdminPortlet.java

```diff
@@ -67,11 +64,6 @@
 
 	@Activate
 	protected void activate(Map<String, Object> properties) {
-		_loginDialogDisabled = GetterUtil.getBoolean(
-			PropsUtil.get(PropsKeys.LOGIN_DIALOG_DISABLED));
-
-		PropsUtil.set(PropsKeys.LOGIN_DIALOG_DISABLED, "true");
```

Comment:
We may need to look at this in more detail. I believe the reason why this property is being set is to prevent a pop-up login from being available, because SAML SSO isn't compatible with it.

FYI  The pop-up normally appears when you comment on something like a blog post whilst logged in as guest, and click "Reply as..." . It enables sign-in without losing the browser viewstate, which enables a JS callback to post the comment following authentication.